site/src/App.jsx

```diff
@@ -7,12 +7,14 @@ import { useState, useEffect, useRef } from "react";
 import Tour from "./Tour";
 import StartupBox from "./StartupBox";
 import { ThemeProvider } from "@mui/material";
+import Sidecar from "./Sidecar";
```

Comment:
Looks like we need a text color alteration for light mode (the labels are still white and should be black) 
<img width="233" alt="image" src="https://github.com/user-attachments/assets/4db81720-7c62-4d97-a867-dd99a47e0a93">